news/2020-19-11-mappers-report-september-october.md

```diff
@@ -0,0 +1,211 @@
+
+You can find the information thread which includes further links and more [here](https://osu.ppy.sh/community/forums/topics/1173347?n=1)!
+
+### Community Guides
+
+This month, we have several mapping and modding related guides to spotlight! The knowledge base the mapping community makes and shares with each other grows ever bigger, and we hope that this can help you find some cool ideas or new content to learn from.
+
+The first is a new YouTube video by the experienced player, mapper, modder, and NAT member [Seto Kousuke](https://osu.ppy.sh/users/2857314)! In his video [5 advices for newbie mappers](https://www.youtube.com/watch?v=zrvfYtYaIAA) he goes over some tips from his experienced perspective that may not be intuitive for beginner mappers to learn, and watching it can be a great way for aspiring mappers to start getting pointed in the right direction in their mapping journey.
+
+The next is a pair of recent videos from long time mapper and Beatmap Nominator [Sonnyc](https://osu.ppy.sh/users/11771). The first, [Making Symmetric Sliders](https://www.youtube.com/watch?v=nYkMFvq4ZLU) serves to quickly show you how to make symmetrical sliders quickly. Even if you don't map symmetrically, knowing how to make balanced sliders to manipulate later can be hugely helpful. The second, [Increasing Mapping Speed](https://www.youtube.com/watch?v=sMKJ9H-qGg0) goes over an experience nearly every mapper has had - mapping slowly at first, and then later how to improve on putting your ideas out there quickly. Other videos on Sonnyc's channel can be super helpful guides to mappers new and old alike, so be sure to check them out!
+
+Moving away from motivation, time taken, and visuals, we have a [guide all about hitsounding](https://hackmd.io/@Net0/HitsoundsGuide) by [Net0](https://osu.ppy.sh/users/5099768)! This guide about hitsounds is incredibly in depth, going all the way from how to name your files to theory, choosing samples, using the editor, and how hitsounding relates back to actual instruments in the songs we love to map and play. If you've ever found hitsounding beatmaps to be intimidating, this guide will definitely help to clear up a lot of questions about it you may have.
+
+Finally, on a bit of a different flavor we have a guide not related to mapping - but modding! The [Modding Mentorship](https://sites.google.com/view/modding-guide/home?authuser=0) guide written by long time mapper and nominator [Cheri](https://osu.ppy.sh/users/5226970) seeks to help give guidance on how to mod and what to look out for from the ground up. This guide covers everything from ranking criteria, to rhythm, to spacing, to low difficulties, and spread. It covers quite a lot! In depth modding guides of this kind are a bit more rare and left to people often learning on their own, so having a fully fleshed out guide like this will definitely be really helpful to aspiring and semi-experienced modders alike.
```

Comment:
```suggestion
Finally, on a bit of a different flavor we have a guide not related to mapping - but modding! The [Modding Mentorship Guide](https://sites.google.com/view/modding-guide/home?authuser=0) written by long time mapper and nominator [Cheri](https://osu.ppy.sh/users/5226970) seeks to help give guidance on how to mod and what to look out for from the ground up. This guide covers everything from ranking criteria, to rhythm, to spacing, to low difficulties, and spread. It covers quite a lot! In-depth modding guides of this kind are a bit rarer and left to people often learning on their own, so having a fully fleshed out guide like this will definitely be really helpful to aspiring and semi-experienced modders alike.
```
'guide' can be included in the hyperlink, since 'Modding Mentorship' does not mean anything alone